app/main/views/frameworks.py

```diff
@@ -115,20 +115,20 @@ def framework_submission_lots(framework_slug):
             'body': lot['description'],
             'statuses': [
                 {
-                    'message': '{} complete service{} {} submitted'.format(
+                    'title': '{} complete service{} {} submitted'.format(
                         count_drafts_by_lot(complete_drafts, lot['value']),
                         '' if 1 == count_drafts_by_lot(complete_drafts, lot['value']) else 's',
                         'was' if 1 == count_drafts_by_lot(complete_drafts, lot['value']) else 'were'
-                    ),
-                    'important': True
-                } if count_drafts_by_lot(complete_drafts, lot['value']) else {},
+                    )
+                } if count_drafts_by_lot(complete_drafts, lot['value']),
```

Comment:
@TheDoubleK You're right. I have acquiesced to Python’s demands and squashed it into 7b8d8a3.